libpseudodev/pseudodev.c

```diff
@@ -0,0 +1,190 @@
+/*
+ * Phoenix-RTOS
+ *
+ * pseudo devices
+ *
+ * Copyright 2023 Phoenix Systems
+ * Author: Gerard Swiderski
+ *
+ * %LICENSE%
+ */
+
+#include <errno.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/msg.h>
+#include <sys/file.h>
+#include "pseudodev.h"
+
+
+static struct {
+	oid_t oid;
+	unsigned int randSeed;
+} pseudo_common;
+
+
+static int pseudo_open(int id, int flags)
```

Comment:
Please note that oid->id is not int. It's platform dependent (at least for now): https://github.com/phoenix-rtos/libphoenix/blob/2a948484a6414a9363c6c33646d517f26d2b9d73/include/arch/armv7/types.h#L27C1-L27C1

Best solution imho is to pass whole oid here to make code oid-id-type-agnostic

Reply:
Right. I'll add a translator for `pseudo_handleMsg()` per multidrv implementation, but I'll keep the `id` as an `int` (it is treated here as a minor index of the device, not `id_t`).

Reply:
Done.